helm/nessie/templates/service.yaml

```diff
@@ -37,6 +37,9 @@ spec:
       port: {{ .number }}
       targetPort: {{ .number }}
       protocol: TCP
+      {{- if and (or (eq $.Values.service.type "NodePort") (eq $.Values.service.type "LoadBalancer")) .nodePort }}
+      nodePort: {{ .nodePort }}
```

Comment:
Do you mind mentioning this new field in values.yaml? Something like below:

```yaml
  ports:
    - # -- The name of the port. Required.
      name: nessie-http
      # -- The number of the port. Required.
      number: 19120
      # -- The port on each node on which this service is exposed. Only meaningful when the 
      # service type is NodePort or LoadBalancer. Leave empty to let Kubernetes assign a random port.
      nodePort: ~
```

Once you change values.yaml, please regenerate the docs by running:

```shell
helm-docs --chart-search-root=helm
```